rollbar/__init__.py

```diff
@@ -1202,6 +1202,35 @@ def post_cb(resp):
     d.addCallback(post_cb)
 
 
+def _send_failsafe(message, uuid, host):
+    body_message = ('Failsafe from pyrollbar: {0}. Original payload may be found '
+                    'in your server logs by searching for the UUID.').format(message)
+
+    data = {
+        'level': 'error',
+        'environment': SETTINGS['environment'],
+        'body': {
+            'message': {
+                'body': body_message
+            }
+        },
+        'notifier': SETTINGS['notifier'],
+        'custom': {
+            'orig-uuid': uuid,
```

Comment:
As a best practice, I would use underscores instead of hyphens for field names. They'll be easier to search using RQL.